### DISCUSSION TITLE
Swell Apps early access

### DISCUSSION THREAD

**Discussion by ericingram** [swellstores/community#458] [Announcements]
For much of 2023, our team has been heads down creating an app framework that sets a new standard of power and flexibility for developers. We’re inviting you to be among the first to see how Swell Apps will enable a new level of innovation in the world of ecommerce.

**What can Swell Apps do?**

The framework provides building blocks that go beyond what any SaaS ecommerce platform offers as native-level extension points, and there’s more to come. Apps can:

- Add new collections and fields that can be displayed in dashboard views
- Run async functions triggered by events, HTTP calls, and cron schedules
- Provide dashboard settings for admins to configure app behavior

Swell Apps are configuration-based, deployed in real-time via CLI, and do not require external infrastructure for most use cases. 

If you’d like to join the alpha program, you will need to:

- Have a store with a test environment
- Join an introductory call to learn about the framework
- Provide feedback in Swell’s Discord about the app development experience, framework mechanics, documentation, and bugs
- Bonus points if you’re willing to share your app source code on GitHub!

> **⚠️ Not for production stores**
> The framework is currently in an alpha state. During this period, there may be breaking changes and we do not recommend using it in live production environments.

To request access, email us at [apps-alpha@swell.is](mailto:apps-alpha@swell.is?subject=I'd%20like%20to%20try%20Swell%20Apps%20alpha&body=My%20store%20URL%20is:) with your store URL and a brief description of what you’d like to accomplish.

**Comment by tarikrital** [comment]
Is this open for the active store or the expired too?

**Reply by ericingram** [reply]
The store needs to have an active account

**Reply by tarikrital** [reply]
Thank you, I have access but clear now.
